tubemq-client-twins/tubemq-client-go/codec/codec/serialization.go

```diff
@@ -0,0 +1,119 @@
+package codec
```

Comment:
Need Licensed

Reply:
Thanks for the suggestion, it has been changed